doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -134,6 +134,7 @@ Updated:
 * Changed settings backend from FCB to NVS.
 * Switched to using :ref:`caf_power_manager`.
 * Fixed an issue with generating motion in :ref:`nrf_desktop_motion` (``motion_buttons`` and ``motion_simulated``) while the HID boot protocol was in use.
+* Fixed an issue with :ref:`nrf_desktop_usb_state` and :ref:`nrf_desktop_hids` modules forwarding the HID input reports related to an old protocol after protocol mode change.
```

Comment:
If this fix is mean for 1.7.0, please add this note in comment to this PR: #5534 and remove the changes to changelog from this file.

Reply:
This issue was present in all versions until the fix. When fix is merged it won't be present anymore.
I think it should be part of known issues and change log for this version.

@alstrzebonski , let's do as @greg-fer said. Get doc changes out of this PR and submit them as separate PRs.